elastic/kibana
# [Synthetics] push pending monitors to bottom of list

## Summary

Resolves https://github.com/elastic/kibana/issues/149429

<img width="1542" alt="Screen Shot 2023-02-06 at 10 03 02 AM" src="https://user-images.githubusercontent.com/11356435/217006824-e39ab668-d1d8-4ace-b918-cdd3c225ae8f.png">

### Testing

1. Create a monitor that will remain in pending state. Ideally, the monitors name should be first alphabetically, so you can have a valid test that the monitor is pushed to the end despite its alphabetical placement. You can ensure it will remain in pending state by doing one of the following: create a private location without attaching it to an elastic-agent or connecting to the dev synthetics service and choosing the staging location.
2. Navigate to Overview. Ensure the pending monitor is pushed to the last in the list
